apps/aefate/src/aefa_engine_state.erl

```diff
@@ -126,6 +161,7 @@
             , stores            :: aefa_stores:store()
             , trace             :: list()
             , vm_version        :: non_neg_integer()
+            , debug_info        :: disabled | debug_info()
```

Comment:
Try to make existence of this field conditional by a preprocessor directive `debug_enabled`

Reply:
I'd keep this as it as since there is no east way of having a conditional field and I don't want to duplicate the `es` record.